website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md

```diff
@@ -84,31 +82,25 @@ The global attributes in the Loom apply to the whole file, not any specific part
-
-
+|`n_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads associated with this entity. |
+|`noise_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads that are categorized by 10x Genomics Cell Ranger as "noise". Refers to long polymers, or reads with high numbers of N (ambiguous) nucleotides. |
+|`perfect_molecule_barcodes`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads with molecule barcodes that have no errors. |
+|`reads_mapped_uniquely`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads mapped to a single unambiguous location in the genome. |
+|`reads_mapped_multiple`|[warp-tools](https://github.com/broadinstitute/warp-tools)|The number of reads mapped to multiple genomic positions with equal confidence. |
+|`spliced_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads that overlap splicing junctions. |
+|`antisense_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads that are mapped to the antisense strand instead of the transcribed strand. |
+|`molecule_barcode_fraction_bases_above_30_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The average fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity. |
+|`molecule_barcode_fraction_bases_above_30_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The variance in the fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity. |
+|`genomic_reads_fraction_bases_quality_above_30_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The average fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity (included for 10x Cell Ranger count comparison). |
+|`genomic_reads_fraction_bases_quality_above_30_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The variance in the fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity (included for 10x Cell Ranger count comparison). |
+|`genomic_read_quality_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Average quality of Illumina base calls in the genomic reads corresponding to this entity. |
+|`genomic_read_quality_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Variance in quality of Illumina base calls in the genomic reads corresponding to this entity. |
+|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity. See README.md for the definition of a Molecule. |
```

Comment:
|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |